src/main/scala/org/broadinstitute/dsde/firecloud/model/Workspace.scala

```diff
@@ -30,3 +30,36 @@ case class EntityCreateResult(
   namespace: String,
   name: String,
   theJSON: String)
+
+
+//swagger annotation
+@ApiModel(value="method configuration entity")
+//scala definitions
+case class MethodConfigurationEntity(
```

Comment:
I guess you mean Greg rename "MethodConfigurationEntity" to just "MethodConfiguration" ?